### PR TITLE
Update base-de.yaml

### PR DESCRIPTION
**Pull request by DerFeldspatz**
Translate recent updates. Also took in the previous shop description, because it was already adapted to the recent changes

### CODE REVIEW

File: translations/base-de.yaml [hunk 1]
```diff
@@ -33,57 +33,52 @@ steamPage:
...
+        Da die Nachfrage sowohl in der Komplexität, als auch der Menge steigt, wirst du deine Fabriken erweitern müssen. Vergiss nicht, dass du die dafür benötigten Ressourcen beschaffen musst und expandiere auf der [b]unendlichen Karte[/b]!
+        Bald wirst du Farben mischen und deine Formen damit bemalen lassen. Staple dann deine fertigen Formen aufeinander und lasse so die wildesten Kreationen entstehen.
+        Nutze dein gesammeltes Wissen über die Maschinen und lasse deine Fabriken die gewünschten Formen der 18 verschiedenen Level abliefern. Schalte mit jedem Level neue Arbeitsschritte oder Gebäude frei. Das sollte dich schon für Stunden beschäftigt halten! Danach werden im Freispielmodus zufällige Formen generiert, die du ebenfalls abliefern kannst. Ich füge regelmäßig neue Funktionen hinzu und davon sind eine ganze Menge geplant!
+        
+        Wenn du das Spiel erwirbst, erhälst du Zugriff auf die zusätzlichen Features der Standalone-Version. Das bedeutet, du kannst unter anderem die neuesten Updates zuerst spielen!
+        [h1]VORTEILE DER STANDALONE[/h1]
```
Comment (MizardX) [inline translations/base-de.yaml:42]: Please do not change the tags that are used. If there is a `[b]...[/b]` in the original, use a `[b]...[/b]` in the translation also.

```
$ node sync-translations.js
...
Processing base-de.yaml
 | Mismatching placeholders in /steamPage/longText -> [
  'img',  'b',
  'b',    'list',
  'b',    'list',
  'b',    'b',
  'list'
] vs [
  'img',  'b',    'h1',
  'list', 'list', 'h1',
  'list', 'h1',   'h1',
  'list'
]
...
```

Comment (DerFeldspatz) [reply translations/base-de.yaml]: Why are b and list placeholders if it's a store description?

Comment (MizardX) [reply translations/base-de.yaml]: It's markdown code. It is probably either uploaded as-is to Steam, or compiled to HTML. The code for uploading to Steam is not included in the repository, so I can't say for sure.

The tags still doesn't match.
```
$ node sync-translations.js
...
Processing base-de.yaml
 | Mismatching placeholders in /steamPage/longText -> [
  'img',  'b',
  'b',    'list',
  'b',    'list',
  'b',    'b',
  'list'
] vs [
  'img',  'b',    'b',
  'list', 'list', 'b',
  'list', 'b',    'b',
  'list'
]
...
```

File: translations/base-de.yaml [hunk 2]
```diff
@@ -33,57 +33,50 @@ steamPage:
...
         [list]
-            [*] Dark Mode
-            [*] Unlimited Waypoints
-            [*] Unlimited Savegames
-            [*] Additional settings
-            [*] Coming soon: Wires & Energy! Aiming for (roughly) end of July 2020.
-            [*] Coming soon: More Levels
-            [*] Allows me to further develop shapez.io ❤️
+            [*] Dark-Mode
+            [*] So viele Wegpunkte wie du willst
```
Comment (tobspr) [inline translations/base-de.yaml:45]: Warum nicht einfach "Unbegrenzte Wegpunkte"?

Comment (tobspr) [reply translations/base-de.yaml:45]: Könntest du auch darauf schauen, dass du die Zeilenumbrüche in der Steam-Beschreibung so beibehältst?

**Comment by tobspr** [comment]
Could you have a look at the conflicts?

**Comment by DerFeldspatz** [comment]
Build failed due to hungarian translation, not mine

**Comment by tobspr** [comment]
Danke! :)

**Pull request merged**